cli/src/klio_cli/commands/job/utils/templates/dockerfile.tpl

```diff
@@ -46,7 +46,11 @@ COPY __init__.py \
      /usr/src/app/
 
 {% if not klio.use_fnapi -%}
+ARG KLIO_CONFIG=klio-job.yaml
+COPY $KLIO_CONFIG klio-job-run-effective.yaml
+
 RUN pip install .
-{% endif -%}
+{%- else -%}
 ARG KLIO_CONFIG=klio-job.yaml
-COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml
\ No newline at end of file
+COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml
+{% endif -%}
```

Comment:
I'm curious why you've added `else` here. Both if/else branches have the same `ARG`/`COPY` lines - I feel like this should have just changed the name in the `COPY` line; or am I missing something?

Reply:
ah yeah it shouldn't have the `ARG` line